python/py-awscli2/Portfile

```diff
@@ -6,11 +6,11 @@ PortGroup           select 1.0
 PortGroup           github 1.0
 
 name                py-awscli2
-github.setup        aws aws-cli 2.19.1
+github.setup        aws aws-cli 2.22.1
 revision            0
 
 categories          python devel
-platforms           {darwin any}
+platforms           {darwin >= 11}
```

Comment:
If it supports macOS 11 and above this is bot correct. The Darwin version is not the same as the macOS version. 
See: https://trac.macports.org/wiki/XcodeVersionInfo

Reply:
Oh, I was not aware of that, thank you. Just updated the PR.